queries.sql

```diff
@@ -7,4 +7,33 @@ SELECT date_of_birth FROM Animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM Animals WHERE weight_kg > 10.5;
 SELECT * FROM Animals WHERE neutered = TRUE;
 SELECT * FROM Animals WHERE name<> 'Gabumon';
-SELECT * FROM Animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM Animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+BEGIN;
+DELETE FROM Animals
+WHERE date_of_birth > '2022-01-01';
+SAVEPOINT weight_update;
+UPDATE Animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO weight_update;
+UPDATE Animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+
```

Comment:
- [ ] You missed a large portion of the query task asked. I've screenshotted it below to show you whats missing. Kindly carry it out, and don't forget add all screenshots to the PR you generated 👍 

### MISSED QUERY:
![image](https://github.com/mumbereh/Vet_Clinic/assets/50990925/d7d91f63-5fbc-4688-8fea-52a3b54ad983)